core/frontend/src/Viewport.ts

```diff
@@ -790,6 +790,11 @@ export abstract class Viewport implements IDisposable, TileUser {
   /** @internal */
   public getMapLayerImageryProvider(index: number, isOverlay: boolean): MapLayerImageryProvider | undefined { return this._mapTiledGraphicsProvider?.getMapLayerImageryProvider(index, isOverlay); }
 
+  /** @beta
+   * Fully reset a map-layer tile tree; by doing call this, the map-layer will to go through initialize process again, and all previously fetched tile will be lost.
```

Comment:
I suppose you mean "this call" instead of "call this"

Reply:
fixed